setup.py

```diff
@@ -0,0 +1,70 @@
+import os
+from glob import glob
+from setuptools import setup, find_packages
+from distutils.command.build import build
+
+
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+
+
+def etc_files():
+    return [(d, [os.path.join(d,f) for f in files])
+            for d, folders, files in os.walk('etc')]
+
+
+def find_scripts():
+    for candidate in glob('bin/*'):
+        with open(candidate) as handle:
+            if handle.readline().startswith("#!"):
+                yield candidate
+
+
+class NAVBuild(build):
+    sub_commands = [
+        ('build_sass', None),
+    ] + build.sub_commands
+
+
+setup(
```

Comment:
This is old-style `setup.py`. New-style is moving all this to `setup.cfg` and just have 
```
from setuptools import setup

setup()
```

in the `setup.py`.

Reply:
Aye, but how does that work when we need to customize parts of the build with our own code?